backend/include/backend/semantizer/semantizer_context.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <list>
+
+#include "error_buffer.hpp"
+#include "semantizer_error.hpp"
```

Comment:
![15197395](https://user-images.githubusercontent.com/43135873/169792730-adfa66dd-0040-43a2-84a9-1f0945c5279b.jpg)

```suggestion
#include "semantizer_error.hpp"

```